web/app_dev.php

```diff
@@ -12,7 +12,7 @@
 // Feel free to remove this, extend it, or make something more sophisticated.
 if (isset($_SERVER['HTTP_CLIENT_IP'])
     || isset($_SERVER['HTTP_X_FORWARDED_FOR'])
-    || !(in_array(@$_SERVER['REMOTE_ADDR'], ['127.0.0.1', 'fe80::1', '::1']) || php_sapi_name() === 'cli-server')
+    || !(in_array(@$_SERVER['REMOTE_ADDR'], ['127.0.0.1', '172.18.0.1', 'fe80::1', '::1']) || php_sapi_name() === 'cli-server')
```

Comment:
Same "hardcoding" policy that for `127.0.0.1`, it's a default value for docker network brigde on a private subnet.
The purpose is to allow docker user to directly access `app_dev.php` without having to modify it.

Could also be replaced with documentation.